packages/manager/modules/sign-up/src/form/activity/translations/Messages_fr_FR.json

```diff
@@ -9,6 +9,10 @@
   "sign_up_activity_field_vat_personalcorporation_more": "Numéro d'identification d'entreprise ({{ vatLabel }})",
   "sign_up_activity_field_organisation_association": "Nom de l'association",
   "sign_up_activity_field_organisation_corporation": "Nom de l'entreprise",
+  "sign_up_activity_corporation_created_true_title": "Mon entreprise est créée",
+  "sign_up_activity_corporation_created_false_title": "Mon entreprise est en cours de création",
+  "sign_up_activity_corporation_created_true_description": "Je remplirai les infos à la création de mon entreprise",
+  "sign_up_activity_corporation_created_false_description": "J'ai mon n° SIRET/SIREN",
```

Comment:
**nitpick:** `true` and `false` in the message keys doesn't make sense to me. Better to use `created` and `not_created`

Reply:
This was a guillaume's suggestion so only one select-picker is created. Should I go back to the previous state (two select pickers with a ng-if and the created / not _created labels) ?

Reply:
Yes I think it is better for better readability